Makefile

```diff
@@ -12,28 +12,41 @@ test: ## run all unit tests
 gofmt: ## format all go files
 	gofmt -l -s -w .
 
-govet:
-	go vet -methods=false .
+govet: ## run go's code vetting on all code
+	go vet ./...
 
-errcheck:
-	go get github.com/kisielk/errcheck@v1.2.0
```

Comment:
So did we loose the errcheck? or is it happening as part of `golangci-lint run` ?

Reply:
It's happening as part of golangci-lint run (it is an explicitly enabled rule):
![image](https://user-images.githubusercontent.com/323009/111674361-27c7f180-87d9-11eb-99e9-23b8f1502dc0.png)